internal/state/store_test.go

```diff
@@ -28,10 +29,16 @@ const (
 func TestStoreBootstrap(t *testing.T) {
 	stateDB := dbm.NewMemDB()
 	stateStore := sm.NewStore(stateDB)
-	val, _ := factory.RandValidator(true, 10)
-	val2, _ := factory.RandValidator(true, 10)
-	val3, _ := factory.RandValidator(true, 10)
-	vals := types.NewValidatorSet([]*types.Validator{val, val2, val3})
+	val, _ := factory.RandValidator()
+	val2, _ := factory.RandValidator()
+	val3, _ := factory.RandValidator()
+	vals := types.NewValidatorSet(
+		[]*types.Validator{val, val2, val3},
+		val.PubKey,
```

Comment:
This won't work, you will need to create the validator set first with 3 validators, as the threshold public key can not just be the first validator public key.